examples/init.rs

```diff
@@ -1,5 +1,7 @@
+use simple_logger::SimpleLogger;
+
 fn main() {
-    simple_logger::init().unwrap();
+    SimpleLogger::new().init().unwrap();
```

Comment:
This makes me wonder. Is the pull request backwards compatible?

Reply:
Yes it is. The previous function are still there, using the builder pattern under the hood.